src/internal/observable/concat.ts

```diff
@@ -139,5 +139,5 @@ export function concat<R>(...observables: (ObservableInput<any> | SchedulerLike)
  * @owner Observable
  */
 export function concat<O extends ObservableInput<any>, R>(...observables: Array<O | SchedulerLike>): Observable<ObservedValueOf<O> | R> {
```

Comment:
Do we really need this `R` here? It seems that it's only present so that folk can explicitly provide it as a type parameter and do we really want to encourage that?